Frontend/src/services/accountServices.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { api_url } from "@/const.js";
+
+let config = {
```

Comment:
still need to look for an unauthorized response and trigger logout on receiving the response. Having the catch redirect statement is a ghetto way of doing it, instead we should trigger the logout function in authorizationServices, which deletes the token from the Server, clears local storage, and redirects you

Reply:
The logout route still needs some work since it hasn't changed since when I did it in our call, but that's ryan's task